### PR TITLE
Added reply feature on front end

### PR DESCRIPTION
**Pull request by kristen-m**

### CODE REVIEW

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -53,14 +53,53 @@ function makeUglySite() {
...
   });
 }
 
 function getLimit() {
-  const limit = document.getElementById("limit").value;
+  const limit = document.getElementById('limit').value;
   return limit;
 }
 
+function createDivElement(text, className) {
```
Comment (damargulis) [inline portfolio/src/main/webapp/script.js:82]: Nice helpers!

File: portfolio/src/main/webapp/style.css [hunk 2]
```diff
@@ -85,7 +85,11 @@ h1 {
 }
 
 #bread {
-    cursor: pointer;
+  cursor: pointer;
+}
+
+#comment-container {
```
Comment (damargulis) [inline portfolio/src/main/webapp/style.css]: no need to include an empty selector

**Review by damargulis — APPROVED** [review]

File: portfolio/src/main/webapp/script.js [hunk 3]
```diff
@@ -53,14 +53,53 @@ function makeUglySite() {
...
+  const divElement = document.createElement('div');
+  divElement.classList = className;
```
Comment (Kaleab-Gebremichael) [inline portfolio/src/main/webapp/script.js]: Instead of overriding the value like this, I think you can use the method, .classList.add(className)

**Review by Kaleab-Gebremichael — APPROVED** [review]
Looks good overall

File: portfolio/src/main/webapp/script.js (hunk 3 above)
Comment (juliareichel) [reply portfolio/src/main/webapp/script.js]: ^ yes similar to the example I posted earlier this morning

File: portfolio/src/main/webapp/style.css (hunk 2 above)
Comment (kristen-m) [reply portfolio/src/main/webapp/style.css]: fixed

File: portfolio/src/main/webapp/script.js (hunk 3 above)
Comment (kristen-m) [reply portfolio/src/main/webapp/script.js]: got it

**Pull request merged**